lib/health_checks/checks/redis_check.rb

```diff
@@ -6,6 +6,7 @@ def initialize(config)
       end
 
       def run
+        @config[:connect_timeout] = 1
```

Comment:
Why only connection timeout? Set it for all operations using `:timeout`

Reply:
because ping checks if the connection is still alive, so I thought it should be enough.